src/components/Search/SearchBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MoviesList } from '../MoviesList';
+import SearchInput from './SearchInput';
+
+class Search extends React.PureComponent {
+  state = {
+    query: '',
+  }
+
+  searchMovie = (event) => {
+    this.setState({
+      query: event.target.value,
+    });
+  }
+
+  filteredMovies = (movies, query) => (
+    movies
+      .filter(movie => movie.title.toLocaleLowerCase()
+        .includes(query.toLocaleLowerCase()))
+    || movies.filter(movie => movie.description.toLocaleLowerCase()
+      .includes(query.toLocaleLowerCase()))
```

Comment:
В одному методі робимо дві перевірки
```suggestion
      .filter(movie => (
        movie.title.toLocaleLowerCase().includes(query.toLocaleLowerCase()
        || movie.description.toLocaleLowerCase().includes(query.toLocaleLowerCase()
      ))
```